source/lib-intro.tex

```diff
@@ -1061,55 +1061,44 @@
 
 
 \pnum
-The facilities of the C standard library are provided in 26
-\indextext{library!C standard}%
-additional headers, as shown in Table~\ref{tab:cpp.c.headers}.%
+The facilities of the C standard library\indextext{library!C standard}
+are provided in 22 additional headers, as shown in Table~\ref{tab:cpp.c.headers}.%
```

Comment:
I thought we were trying to avoid keeping accurate counts, as they are too much effort to maintain? Prefer just 'additional headers' without the 22?